src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -1,75 +1,84 @@
 import React, { useState, useEffect } from 'react'
 import styled from '@emotion/styled'
 import { colorsV3 } from '@hedviginsurance/brand'
-import { Address } from 'data/graphql'
 import { OfferData } from 'pages/OfferNew/types'
 import { useTextKeys } from 'utils/textKeys'
-import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
-import { getHouseholdSize } from '../utils'
+import {
+  LARGE_SCREEN_MEDIA_QUERY,
+  MEDIUM_SCREEN_MEDIA_QUERY,
+} from 'utils/mediaQueries'
+import { getHouseholdSize, quoteDetailsHasAddress } from '../utils'
+import { getAddress } from '../Checkout/InsuranceSummaryDetails'
 
 type Props = {
   offerData: OfferData
 }
 
-type QuoteWithStreet = {
-  quoteDetails: { street: Address['street'] }
-}
-
-const Container = styled.div`
-  padding: 2rem 0 1rem;
+const Wrapper = styled.div`
+  padding: 2.5rem 0 1rem;
   color: ${colorsV3.white};
 
   ${LARGE_SCREEN_MEDIA_QUERY} {
     padding-top: 0;
   }
 `
-
-const Heading = styled.h1`
+const Headline = styled.h1`
   margin: 0;
   text-transform: uppercase;
-  font-size: 0.75rem;
+  font-size: 1rem;
+`
+const OfferInfoWrapper = styled.div`
+  padding: 1rem 0;
```

Comment:
Hm actually I think it might look nicer with 
```suggestion
  padding-bottom: 2rem;
```
![image](https://user-images.githubusercontent.com/42962286/124887056-1e0b0980-dfd5-11eb-9b18-9bce198f54be.png)